README.md

```diff
@@ -9,6 +9,7 @@
 [![Contributors][contributors-shield]][contributors-url]
 [![Stargazers][stars-shield]][stars-url]
 [![Issues][issues-shield]][issues-url]
+[![Sonarcloud Status](https://sonarcloud.io/api/project_badges/measure?project=smallbusinesshero_dds-wvv-frontend&metric=alert_status)](https://sonarcloud.io/api/project_badges/measure?project=smallbusinesshero_dds-wvv-frontend&metric=alert_status)
```

Comment:
I would suggest adding the Link to the project in sonarcloud, instead of the link to the badge
https://sonarcloud.io/dashboard?id=smallbusinesshero_dds-wvv-frontend
other than that everything seems to work

Reply:
Done